test/test.js

```diff
@@ -491,6 +491,19 @@ suite('Request', function() {
     })
   })
 
+  test('clone request bodies', function() {
+    var r1 = new Request('https://fetch.spec.whatwg.org/', {
+      method: 'post',
+      headers: {'content-type': 'text/plain'},
+      body: 'I work out'
```

Comment:
Ah I see what you mean. That makes sense to do. I'm writing a test now although it seems to 'just work' so I just need to make sure it's testing properly...